js/main.js

```diff
@@ -103,7 +103,7 @@ requirejs.config({
 		"negative-controls": "components/negative-controls",
 		"d3": "d3.min",
 		"d3_tip": "d3.tip",
-		"jnj_chart": "jnj.chart",
+		"jnj_chart": "https://cdn.rawgit.com/odysseusinc/atlascharts/master/index",
```

Comment:
@chrisknoll, @t-abdul-basser  - agree with all your comments. This module was meant to stay in OHDSI repo. With all good intentions of creating a re-usable module, we should have done a better job in discussing everything in this group first - lesson learned.